wolfcrypt/src/asn.c

```diff
@@ -11075,6 +11172,17 @@ int wc_MakeSelfCert(Cert* cert, byte* buffer, word32 buffSz,
 
 #ifdef WOLFSSL_CERT_EXT
 
+/* Get raw subject from cert, which may contain OIDs not parsed by Decode. */
+int wc_GetSubjectRaw(byte **subjectRaw, Cert *cert)
```

Comment:
Can you also add some docs for this API in `doc/dox_comments/header_files/asn_public.h`?

Reply:
Got it